lxd/endpoints/endpoints.go

```diff
@@ -305,6 +315,11 @@ func (e *Endpoints) up(config *Config) error {
 		e.serve(metrics)
 	}
 
+	if e.listeners[vsock] != nil {
```

Comment:
If there's no vsock listener, there doesn't seem much point in continuing with lxd-agent?

Reply:
What do you mean? I don't quite understand. The `lxd-agent` doesn't rely on vsock, it only requires it for devlxd.

Reply:
Ah ok yes I think I thought this was the devlxd in the agent.